pandas-stubs/core/series.pyi

```diff
@@ -935,8 +935,8 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     def isin(self, values: Iterable | Series[S1] | dict) -> Series[_bool]: ...
     def between(
         self,
-        left: Scalar | Sequence,
-        right: Scalar | Sequence,
+        left: Scalar | Sequence | Series,
+        right: Scalar | Sequence | Series,
```

Comment:
Can you change this to `Scalar | ListLikeU` ?  `between` accepts  `Index` and `np.ndarray` as well.